sites/hn.svelte.dev/src/routes/[list=category]/[page]/+page.js

```diff
@@ -1,4 +1,3 @@
-/** @type {import('./$types').PageLoad} */
```

Comment:
Why remove the types?

Reply:
Cuz they're automatically inferred by svelte extension, and gives better JSDoc results than manually typing. Its also lesser number of lines

Reply:
Will `svelte-check` automatically infer them as well?

Reply:
It should, svelte.dev doesn't have typings on these functions and lint step passes there

Reply:
Well we don't run `svelte-check` as far as I can see, so that doesn't prove much :laughing: Plus, I don't think it would error on an `any` type - only a known type that was wrong.

@dummdidumm do you know if `svelte-check` is able to infer these types?

Reply:
Just added svelte check and ran it. Works fine here.

AFAIK, any type is not allowed with noImplicitAny, which is on for this project. So svelte-check os detecting the types automatically methinka

Reply:
Ah got it! checkJS needs to be true